viral/run_oasis.py

```
@@ -73,216 +75,93 @@ def modwt_denoise(
-            cell[
-                wheel_freeze.post_training_start_frame : wheel_freeze.post_training_end_frame
-            ],
-        ],
-    ):
-        raw = np.append(raw, chunk)
-        chunk_baselined, chunk_baseline = compute_dff_percentile_filter(
-            chunk, percentile=5, window_size_seconds=90
-        )
-        # Grosmark does this but I don't think it changes anything
-        chunk_baselined = chunk_baselined - np.median(chunk_baselined)
+    f = get_f(s2p_path)
```

Ruff (0.13.1)

Undefined name `get_f`

(F821)